### PR TITLE
fix(core): checkbox broken VoiceOver outline

### PR DESCRIPTION
**Pull request by genereg1**
## Related Issue(s)

closes https://github.com/SAP/fundamental-ngx/issues/5791

## Description

fix broken VoiceOver outline in Checkbox component 

## Screenshots

> **NOTE:** If you've made any style changes, please provide appropriate screenshots (before and after) to help reviewers.

### Before:

Screenshot 2021-10-27 at 19 47 35

### After:

Screenshot 2021-10-27 at 19 51 38

##### During Implementation

1. Visual Testing:

-   [x] visual misalignments/updates
-   [x] check Light/Dark/HCB/HCW themes
-   [x] RTL/LTR - proper rendering and labeling
-   [x] responsiveness(resize)
-   [x] Content Density (Cozy/Compact/(Condensed))
-   [x] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [x] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [x] Mouse vs. Keyboard support
-   [x] Text Truncation

2. API and functional correctness

-   [x] check for console logs (warnings, errors)
-   [n\a] API boundary values
-   [n\a] different combinations of components - free style
-   [n\a] change the API values during testing

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality

-   [x] the commit message(s) follows the guideline:
        https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
-   [x] tests for the changes that have been done
-   [x] all items on the PR Review Checklist are addressed :
        https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
-   [x] Run npm run build-pack-library and test in external application
-   [n\a] update `README.md`
-   [n\a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

### CODE REVIEW

File: libs/core/src/lib/checkbox/checkbox/checkbox.component.scss [hunk 1]
```diff
@@ -3,3 +3,11 @@
...
     margin: 0;
 }
+
+.fd-checkbox,
+.fd-checkbox.is-warning,
+.fd-checkbox.is-error,
+.fd-checkbox.is-success,
+.fd-checkbox.is-information {
+    border-style: initial;
+}
```
Comment (InnaAtanasova) [inline libs/core/src/lib/checkbox/checkbox/checkbox.component.scss:13]: should go to fund-styles

Comment (genereg1) [reply libs/core/src/lib/checkbox/checkbox/checkbox.component.scss]: raised a PR https://github.com/SAP/fundamental-styles/pull/2881

**Review by platon-rov — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Comment by genereg1** [comment]
waits for https://github.com/SAP/fundamental-styles/pull/2881

**Comment by puru-hk** [comment]
I think still issue is not fixed attached the screenshot
Screenshot 2021-11-19 at 2 12 16 PM

**Comment by genereg1** [comment]
> I think still issue is not fixed attached the screenshot Screenshot 2021-11-19 at 2 12 16 PM
@puru-hk 
Did you check it through Voice Over? It works from my side. Could you record a small video instead of it?

**Comment by puru-hk** [comment]
> https://deploy-preview-7036--fundamental-ngx.netlify.app

Yes I checked in chrome with voiceover. Attached the video. 

https://user-images.githubusercontent.com/32538291/142641131-a8d28961-b500-4c97-9f7d-e5e1d7abbca7.mov

**Comment by puru-hk** [comment]
Its work fine

**Review by puru-hk — APPROVED** [review]

**Pull request merged**